generators/typescript/sdk/generator/src/contexts/type-schema/TypeSchemaContextImpl.ts

```diff
@@ -175,7 +175,7 @@ export class TypeSchemaContextImpl implements TypeSchemaContext {
                         return { type: "fromRoot", useDynamicImport: false, namespaceImport: "serializers" };
                     } else if (isGeneratingSchema) {
                         // Return default import strategy or another strategy based on your logic
-                        return { type: "direct" };
+                        return { type: "direct", alias: `${typeName.name.originalName}Type` };
```

Comment:
See previous comment about line 149, and the aliases being identical. It may be meaningful to extract this into a utility function to make this less prone to breakage. On the other hand, importing the same type twice with different aliases isn't going to break anything, even if it is redundant.